### PR TITLE
Orca Data Image Write API

### PR DESCRIPTION
**Pull request by GZHoffie**
#3909.

### CODE REVIEW

**Comment by yangw1234** [comment]
LGTM

**Comment by yangw1234** [comment]
http://10.239.176.111:18888/job/ZOO-PR-Python-Spark-2.4-py36/4007/console, please fix the coding style error

**Comment by GZHoffie** [comment]
Jenkins passed: http://10.239.176.111:18888/job/ZOO-PR-Validation/5590/

**Comment by yangw1234** [comment]
LGTM

File: pyzoo/zoo/orca/data/image/parquet_dataset.py [hunk 1]
```diff
@@ -246,3 +246,22 @@ def make_generator():
...
     ParquetDataset.write(output_path, make_generator(), schema, **kwargs)
+
+
+def _check_arguments(_format, kwargs, args):
+    for keyword in args:
+        assert keyword in kwargs, keyword + " is not specified for format " + _format + "."
+
+
+def write_parquet(format, output_path, *args, **kwargs):
+    supported_format = {"imagenet", "mnist", "image_folder", "voc"}
```
Comment (yangw1234) [inline pyzoo/zoo/orca/data/image/parquet_dataset.py:257]: need to remove "imagenet"

Comment (GZHoffie) [reply pyzoo/zoo/orca/data/image/parquet_dataset.py:257]: OK, I opened a new PR for this.

**Pull request merged**